core/src/test/java/fr/sncf/osrd/api/InfraLoadingTest.java

```diff
@@ -28,6 +29,20 @@ public void simple() throws Exception {
                 "tiny_infra/infra.json",
                 "1"
         );
-        assertEquals("Infra loaded", runInfraLoad(query));
+
```

Comment:
Test cases are missing: it does not test if the load is done or not, just that the response is ok ; tit does not check that an exception is thrown when the body is empty.